### PR TITLE
Assortment of UI Fixes

### PR DESCRIPTION
**Pull request by prwelber**

### CODE REVIEW

**Comment by prwelber** [comment]
depends on https://github.com/blockchain/brand-assets/pull/3

File: app/partials/export-history.pug [hunk 1]
```diff
@@ -2,7 +2,7 @@ form(role="form" name="exportForm" ng-submit="submit()" autocomplete="off" noval
   .modal-header
     .flex-center.pam
       h3.man.mrm(translate="EXPORT_HISTORY")
-      helper-button(content="EXPORT_HISTORY_EXPLAIN")
+      helper-button.show(content="EXPORT_HISTORY_EXPLAIN")
```
Comment (plondon) [inline app/partials/export-history.pug]: What is .show for?

File: assets/js/controllers/transactions.controller.js [hunk 2]
```diff
@@ -74,7 +74,11 @@ function TransactionsCtrl ($scope, $q, $translate, $uibModal, Wallet, MyWallet,
   $scope.$on('$destroy', unsub);
 
   // Searching and filtering
-  $scope.filterTypes = ['ALL', 'SENT', 'RECEIVED', 'TRANSFERRED'];
+  if (window.innerWidth <= 768) {
```
Comment (plondon) [inline assets/js/controllers/transactions.controller.js]: This should be < 768 or <= 767

File: app/partials/export-history.pug (hunk 1 above)
Comment (prwelber) [inline app/partials/export-history.pug]: Helper buttons are normally hidden at mobile width, but I guess we don't want that to be the case for this one.

File: assets/js/controllers/transactions.controller.js (hunk 2 above)
Comment (jtormey) [inline assets/js/controllers/transactions.controller.js]: Should really avoid dealing with layout logic in controllers, is there any way you could do this with css? If not (or if it's too much of a pain), could you use `$scope.$root.size.xs` instead?

File: assets/js/controllers/transactions.controller.js [hunk 3]
```diff
@@ -74,7 +74,7 @@ function TransactionsCtrl ($scope, $q, $translate, $uibModal, Wallet, MyWallet,
   $scope.$on('$destroy', unsub);
 
   // Searching and filtering
-  if (window.innerWidth <= 768) {
+  if ($scope.$root.size.sm || $scope.$root.size.xs) {
```
Comment (jtormey) [inline assets/js/controllers/transactions.controller.js]: This covers any width less than 992px, intentional? You had 768 before, just making sure.

Comment (plondon) [inline assets/js/controllers/transactions.controller.js]: @prwelber

Comment (prwelber) [inline assets/js/controllers/transactions.controller.js]: The filter bar's breakpoint is 991px, so this works well.

**Pull request merged**